### PR TITLE
fix: use data model for caseType/procedure in mapping

### PR DESCRIPTION
**Pull request by bryn-pins**
## Checklist

- [ ] Requires infrastructure changes
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [ ] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `main` (please only [rebase](https://github.com/Planning-Inspectorate/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

File: packages/appeals-service-api/src/services/back-office-v2/formatters/s78/appeal.js [hunk 1]
```diff
@@ -57,8 +58,8 @@ exports.formatter = async (appellantSubmission) => {
 	return {
 		casedata: {
 			submissionId: appellantSubmission.appealId,
-			caseType: 'D',
-			caseProcedure: 'written',
+			caseType: APPEAL_CASE_TYPE.W,
```
Comment (topickering) [inline packages/appeals-service-api/src/services/back-office-v2/formatters/s78/appeal.js:61]: Almost certainly my ignorance of business logic, but how come the Case Type is W for s78 and D for HAS?

Comment (topickering) [reply packages/appeals-service-api/src/services/back-office-v2/formatters/s78/appeal.js:61]: Does caseType correspond directly to HAS, S78 etc?

Comment (bryn-pins) [reply packages/appeals-service-api/src/services/back-office-v2/formatters/s78/appeal.js:61]: D is always HAS, so was just copied from HAS incorrectly

I'm not sure yet if we need to change the case type for S78 based on the answers provided, but W should be S78 written

Comment (topickering) [reply packages/appeals-service-api/src/services/back-office-v2/formatters/s78/appeal.js:61]: Thanks!

**Review by topickering — APPROVED** [review]

**Pull request merged**